dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -447,19 +456,58 @@ private ExperimentVariant createExperimentVariant(final Experiment experiment, f
         variantAPI.save(Variant.builder().name(variantName)
                 .description(Optional.of(variantDescription)).build());
 
-        APILocator.getMultiTreeAPI().copyVariantForPage(experiment.pageId(),
+        multiTreeAPI.copyVariantForPage(experiment.pageId(),
                 DEFAULT_VARIANT.name(), variantName);
 
         final Contentlet pageContentlet = contentletAPI
                 .findContentletByIdentifierAnyLanguage(experiment.pageId(), false);
 
-        final HTMLPageAsset page = APILocator.getHTMLPageAssetAPI().fromContentlet(pageContentlet);
+        // copy page for variant
+        if(variantDescription.equals(ORIGINAL_VARIANT)) {
```

Comment:
maybe we should move this logic to a method

Reply:
Done